microsoft-365/compliance/sit-edm-wizard.md

```diff
@@ -49,40 +49,55 @@ steps in [Part 1: Set up EDM-based classification](create-custom-sensitive-infor
+2. Choose **Choose an existing EDM schema** and choose the schema you created in steps 2-6 from the list.
 
-11.  Choose the **Primary element's sensitive info type** to associate it with. See [Sensitive Information Type Entity Definitions](sensitive-information-type-entity-definitions.md) to learn more about the available sensitive information types.
+3. Choose **Next** and choose **Create pattern**.
 
-12. Choose **Done**.
+4. Choose the **Confidence level** for this pattern. The confidence level you select is the one that will be assigned to the match when the content in the document meets the criteria you define next. To learn more about configuring a pattern, see [Create a custom sensitive information type in the Compliance Center](create-a-custom-sensitive-information-type.md).
 
-13. Choose your desired **Confidence level and character proximity**.  This will be the default value for the whole EDM sensitive info type
+5. Choose the **Primary element** from the list of columns in your schema. The primary element will be the column in the table that this pattern will be comparing content against anywhere in a document or email. 
 
-13. Choose **Create pattern** if you want to creaet additional patterns for your EDM sensitive info type.
+6.  Choose the **Primary element's sensitive info type** to define what text in the document will be compared with all the values in the column you selected before. For example, if the column you selected contains your customers' social security numbers you can select the built-in U.S. Social Security Number sensitive info type from the list, which will cause EDM to look iinto content for any strings that look like social security numbers, and perform a comparison against all values in the relevant column in your sensitive information table until an exact match is found (minus the ignored delimiters you configured). 
 
-14. Choose **Next** and fill in a **Name** and **Description for admins**.
+7.  If none of the built-in sensitive info types will match the data in the column you selected you will have to create a custom SIT that does. If you selected the Ignored Delimiters option for this column when defining your schema, make sure the custom SIT you create will match data with and without the selected delimiters. Also, avoid creating a custom SIT that would match too much unrelated text (e.g. one that uses a regular expression that matches any random words or string) since this can cause the system to temporariluy stop matching data. See [Sensitive Information Type Entity Definitions](sensitive-information-type-entity-definitions.md) to learn more about the available sensitive information types and [Get started with custom sensitive information types](create-a-custom-sensitive-information-type.md) to learn about creating a custom sensitive infromation type. 
```

Comment:
```suggestion
7.  If none of the built-in sensitive info types will match the data in the column you selected you will have to create a custom sensitive info type that does. If you selected the Ignored Delimiters option for this column when defining your schema, make sure the custom sensitive info type you create will match data with and without the selected delimiters. Also, avoid creating a custom sensitive info type that would match too much unrelated text (such as one that uses a regular expression that matches any random words or string) since this can cause the system to temporarily stop matching data. See [Sensitive Information Type Entity Definitions](sensitive-information-type-entity-definitions.md) to learn more about the available sensitive information types and [Get started with custom sensitive information types](create-a-custom-sensitive-information-type.md) to learn about creating a custom sensitive information type. 
```